src/site/content/en/blog/url-protocol-handler/index.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title: URL protocol handler registration for PWAs
+subhead: |
+  Let installed PWAs handle links that use a specific protocol for a more integrated experience.
+authors:
+  - thomassteiner
+date: 2021-05-05
+description: |
+  After registering a PWA as a protocol handler, when a user clicks on a hyperlink with a specific
+  scheme such as mailto, ms-word, or web+music from a browser or a platform-specific app,
```

Comment:
I was wondering about this, but since it was mentioned in the Explainer, I used it in the article. You might want to update the Explainer accordingly then (or add a remark that this is a potential future addition).